### PR TITLE
Aresite2

### PR DESCRIPTION
**Pull request by jfallmann**
Script for interaction with AREsite2 REST interface from Galaxy

### CODE REVIEW

File: tools/rna_tools/aresite2/aresite2.xml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+<tool id="AREsite2_REST" name="Get motifs from AREsite2" version="0.1.2">
+    <description>AREsite2 REST Interface</description>
+    <requirements>
+        <requirement type="package" version="2.12.4">requests</requirement>
+        <requirement type="package" version="1.10.0">six</requirement>
+    </requirements>
+    <command>
+<![CDATA[
+python $__tool_directory__/aresite2.py
```
Comment (bgruening) [inline tools/rna_tools/aresite2/aresite2.xml]: single quotes

File: tools/rna_tools/aresite2/aresite2.xml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+-g $gene
+-m "$motif"
```
Comment (bgruening) [inline tools/rna_tools/aresite2/aresite2.xml]: single quotes ... reason is that double quotes are evaluated in bash and can cause harm

File: tools/rna_tools/aresite2/aresite2.xml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+-s $species_selector
+> $AREsite2_REST_out
```
Comment (bgruening) [inline tools/rna_tools/aresite2/aresite2.xml]: single quotes

File: tools/rna_tools/aresite2/aresite2.xml [hunk 4]
```diff
@@ -0,0 +1,53 @@
+<tool id="AREsite2_REST" name="Get motifs from AREsite2" version="0.1.2">
+    <description>AREsite2 REST Interface</description>
+    <requirements>
+        <requirement type="package" version="2.12.4">requests</requirement>
+        <requirement type="package" version="1.10.0">six</requirement>
+    </requirements>
+    <command>
+<![CDATA[
+python $__tool_directory__/aresite2.py
+-g $gene
```
Comment (bgruening) [inline tools/rna_tools/aresite2/aresite2.xml]: all text and data parameter should be single quoted

**Comment by bgruening** [comment]
@jfallmann looks great! To make travis test this, you need to include a .shed.yml file next to your tool. This will also make it possible to automatically upload the wrapper to the TS.

**Comment by jfallmann** [comment]
@bgruening Oh, somethings broken here
The command "planemo ci_find_repos --exclude_from .tt_blacklist \

                        --exclude packages --exclude data_managers \

                        --changed_in_commit_range "$TRAVIS_COMMIT_RANGE" \

                        --chunk_count 2 --chunk "${CHUNK}" \

                        --output changed_repositories_chunk.list

  " failed and exited with 1 during .

**Comment by jfallmann** [comment]
@bgruening Thx for the yml fix, the new error is python version related

**Comment by bgruening** [comment]
Ah python2 vs. python3, either you include python=2.7.somthing in your requirement file, or you port your script to be compatible with python3 as well.

**Comment by bgruening** [comment]
Thanks @jfallmann!

**Pull request merged**